backend/geonature/core/users/routes.py

```diff
@@ -138,6 +138,7 @@ def get_role(id_role):
     fields = user_fields.copy()
     if g.current_user == user:
         fields.add("email")
+        fields.add("champs_addi")
```

Comment:
C'est pas mal la protection de ne rajouter les info "confidentielles" uniquement lorsque c'est l'utilisateur lui même qui demande la route. Je rajouterais bien des tests là dessus car ça peut être un beau leak de donnée + commentaire explicite sur le pourquoi on fait ça.
```suggestion
        fields.update({"email", "champs_addi"})
```

Reply:
Tu penses que au travers des champs addi, on puisse récupérer les mots de passes ?